### ISSUE TITLE
Failure to run on Linux

### ISSUE THREAD

**Issue by plg6** [d4software/QueryTree#101]
Good day,

I've got the dotnet installed on this Ubuntu 18.04 based distro, but I think there's a security certificate error which may be the root cause of the not being able to access the Nuget list. First the errors:

/usr/share/dotnet/sdk/2.2.402/NuGet.targets(123,5): error : Unable to load the service index for source https://api.nuget.org/v3/index.json. [/opt/QueryTree/Web/QueryTree.csproj]
/usr/share/dotnet/sdk/2.2.402/NuGet.targets(123,5): error :   The SSL connection could not be established, see inner exception. [/opt/QueryTree/Web/QueryTree.csproj]
/usr/share/dotnet/sdk/2.2.402/NuGet.targets(123,5): error :   The remote certificate is invalid according to the validation procedure. [/opt/QueryTree/Web/QueryTree.csproj]

The build failed. Please fix the build errors and run again.

Now the dotnet install info:

.NET Core SDK (reflecting any global.json):
 Version:   2.2.402
 Commit:    c7f2f96116

Runtime Environment:
 OS Name:     linuxmint
 OS Version:  19.3
 OS Platform: Linux
 RID:         linux-x64
 Base Path:   /usr/share/dotnet/sdk/2.2.402/

Host (useful for support):
  Version: 2.2.8
  Commit:  b9aa1abc51

.NET Core SDKs installed:
  2.2.402 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.2.8 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.2.8 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.2.8 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

Must be some devil in the details, but I sure would like to exorcise this bug and get to using QueryTree as it sounds quite useful. Thanks very much in advance.

**Comment by d4nt** [comment]
@plg6 This looks like it's just a problem with with nuget (i.e. not really anything to do with QueryTree). Possibly something to do with your HTTP proxy settings, see this thread: https://github.com/NuGet/Home/issues/2880

**Comment by plg6** [comment]
Thanks for your reply. I've made some progress. I found that issuing  export DOTNET_SYSTEM_NET_HTTP_USESOCKETSHTTPHANDLER=0 
gets me past the original error I was having, but it now starts to run for a while, but then hangs forever on:
dbug: Hangfire.SQLite.ExpirationManager[0]
      Removing outdated records from the 'Hash' table...

So QueryTree never fully starts. I realize this may not be a totally QueryTree issue, but I am totally brand new to all of this dotnet and am just trying to get this one app running so I can have people access data in a MySQL database whether or Mac or PC. Thanks for your understanding and patience.
